pgjdbc/src/test/java/org/postgresql/test/jdbc2/DatabaseMetaDataPropertiesTest.java

```diff
@@ -64,6 +64,7 @@ public void testCapabilities() throws SQLException {
     // we support multiple resultsets via multiple statements in one execute() now
     assertTrue(dbmd.supportsMultipleResultSets());
 
+    assertTrue(dbmd.supportsTransactions());
```

Comment:
What does this have to do with what we are currently testing ?

Reply:
Just that it's related to transaction support and wasn't already tested. `testCapabilities` felt like the right place for it, but I removed it.